lib/api.js

```diff
@@ -47,7 +48,7 @@ exports.default = function (_ref) {
   if (!module.exports) {
     // babel did not transform modules
     // eslint-disable-next-line no-underscore-dangle
-    component = module.__esModule ? module.default : module;
+    component = ctx.__esModule ? ctx.default : ctx;
```

Comment:
From my tests there was always a `default` property on the object (even if that's all that's being exported), so maybe we could just drop this and test for `default` ? Not sure what to do when there are no default and only named exports ?